card/mod/05_email/set/all/notify.rb

```diff
@@ -68,7 +68,7 @@ def silent_change?
 end
 
 def notable_change?
-  !silent_change? && !supercard && current_act &&
+  !silent_change? && !supercard && current_act && current_act.card_id == id &&
```

Comment:
It's not really wrong. `!supercard` in the same line is supposed to do something similar (or the same) namely ensure that only one email per act is send. But it might be not the right way to do it.
Looks like the current behaviour is: We send an email to a user if the act card is followable and there was change to a card that is part of the act and followed by the user.